src/js/views/programs/sidebar/program/programs-sidebar_views.js

```diff
@@ -142,10 +142,12 @@ const LayoutView = View.extend({
     this.program = program;
     this.model = this.program.clone();
 
-    this.listenTo(this.program, {
-      'change:published_at': this.showPublished,
-      'change:archived_at': this.showArchived,
-    });
+    if (!this.model.isNew()) {
+      this.listenTo(this.program, {
+        'change:published_at': this.showPublished,
+        'change:archived_at': this.showArchived,
+      });
+    }
```

Comment:
This fixes a bug where a console error is thrown after successfully creating a new program. Which is caused by the listener I added in a previous commit.

What was happening:

1. Open new program sidebar. At this time, the `[data-published-region]` and `[data-archived-region]` regions are hidden because `isNew() === true`.
2. Hit `Save` button to create a new program. When the `POST` request comes back, the `change:published_at` and `change:archived_at` listeners are triggered. Which means `showPublished()` and `showArchived()` try to do a render on those `[data-published-region]` and `[data-archived-region]` regions. `isNew()` now equals `false`, but those regions don't exist yet. And this all happens before the program sidebar closes.

So to avoid this, I only add the listeners if the model is not new.